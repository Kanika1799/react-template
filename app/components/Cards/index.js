/**
 *
 * Cards
 *
 */

import React, { memo } from 'react';
// import PropTypes from 'prop-types'

import { FormattedMessage as T } from 'react-intl';

function Cards() {
  return (
    <div data-testid="cards">
      <T id={'cards'} />
      cards.map {}
    </div>
  );
}

Cards.propTypes = {};

export default memo(Cards);
