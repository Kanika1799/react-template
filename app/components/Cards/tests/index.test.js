/**
 *
 * Tests for Cards
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom'
import { renderWithIntl } from '@utils/testUtils';
import Cards from '../index';

describe('<Cards />', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = renderWithIntl(<Cards />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should contain 1 Cards component', () => {
    const { getAllByTestId } = renderWithIntl(<Cards />);
    expect(getAllByTestId('cards').length).toBe(1);
  });
});
