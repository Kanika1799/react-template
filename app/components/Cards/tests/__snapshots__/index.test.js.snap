// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards /> should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="site-card-wrapper"
      data-testid="cards"
    >
      <div
        class="ant-row"
        style="margin-left: -10px; margin-right: -10px;"
      >
        <div
          class="ant-col ant-col-4"
          style="padding-left: 10px; padding-right: 10px;"
        >
          <div
            class="ant-card ant-card-bordered Cards__IconCard-a1oo2v-0 hJopCh"
          >
            <div
              class="ant-card-body"
            >
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  class="ant-col"
                >
                  <h3
                    class="Cards__Title-a1oo2v-1 dzKJiK"
                  >
                    2345
                  </h3>
                  2345
                  <h4
                    class="Cards__Heading-a1oo2v-2 kICSuH"
                  >
                    INVOICES ADDED
                  </h4>
                  <a
                    class="Cards__Link-a1oo2v-3 bKZrrg"
                    href="#"
                  >
                    ADD INVOICE 
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-col ant-col-4"
          style="padding-left: 10px; padding-right: 10px; margin-left: 50px;"
        >
          <div
            class="ant-card ant-card-bordered Cards__IconCard-a1oo2v-0 hJopCh"
          >
            <div
              class="ant-card-body"
            >
               
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  class="ant-col"
                >
                  <h3
                    class="Cards__Title-a1oo2v-1 dzKJiK"
                  >
                    05
                  </h3>
                  <h4
                    class="Cards__Heading-a1oo2v-2 kICSuH"
                  >
                    BANK ACCOUNT ADDED
                  </h4>
                  <a
                    class="Cards__Link-a1oo2v-3 bKZrrg"
                    href="#"
                  >
                    ADD BANK ACCOUNT
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-col ant-col-4"
          style="padding-left: 10px; padding-right: 10px; margin-left: 50px;"
        >
          <div
            class="ant-card ant-card-bordered Cards__IconCard-a1oo2v-0 hJopCh"
          >
            <div
              class="ant-card-body"
            >
               
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  class="ant-col"
                >
                  <h3
                    class="Cards__Title-a1oo2v-1 dzKJiK"
                  >
                    15
                  </h3>
                  <h4
                    class="Cards__Heading-a1oo2v-2 kICSuH"
                  >
                    UNVERIFIED INVOICES
                  </h4>
                  <a
                    class="Cards__Link-a1oo2v-3 bKZrrg"
                    href="#"
                  >
                    VERIFY INVOICE 
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-col ant-col-4"
          style="padding-left: 10px; padding-right: 10px; margin-left: 50px;"
        >
          <div
            class="ant-card ant-card-bordered Cards__IconCard-a1oo2v-0 hJopCh"
          >
            <div
              class="ant-card-body"
            >
               
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  class="ant-col"
                >
                  <h3
                    class="Cards__Title-a1oo2v-1 dzKJiK"
                  >
                    03
                  </h3>
                  <h4
                    class="Cards__Heading-a1oo2v-2 kICSuH"
                  >
                    TRANSACTIONS FAILED
                  </h4>
                  <a
                    class="Cards__Link-a1oo2v-3 bKZrrg"
                    href="#"
                  >
                    VIEW DETAILS
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-col ant-col-4"
          style="padding-left: 10px; padding-right: 10px; margin-left: 50px;"
        >
          <div
            class="ant-card ant-card-bordered Cards__IconCard-a1oo2v-0 hJopCh"
          >
            <div
              class="ant-card-body"
            >
               
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  class="ant-col"
                >
                  <h3
                    class="Cards__Title-a1oo2v-1 dzKJiK"
                  >
                    $2345.39
                  </h3>
                  <h4
                    class="Cards__Heading-a1oo2v-2 kICSuH"
                  >
                    UPCOMING PAYMENT
                  </h4>
                  <a
                    class="Cards__Link-a1oo2v-3 bKZrrg"
                    href="#"
                  >
                    VIEW INVOICE 
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
